ads/google/adsense.js

```diff
@@ -39,6 +39,11 @@ export function adsense(global, data) {
       ADSENSE_RSPV_WHITELISTED_HEIGHT +
       ' for responsive AdSense ad units.');
 
+  user().assert(
```

Comment:
Now that we have 2 different rspv specific error states to check, I think this will be more readable if we wrap in if:

if (data['autoFormat']) {
   user().assert(data['height'] == ...);
   user().assert(data['width'] == ...);
}